op_crates/web/11_url.js

```diff
@@ -37,7 +37,15 @@
   }
 
   function decodeSearchParam(p) {
-    return decodeURIComponent(p.replace(/\+/g, " "));
+    const s = p.replace("+", " ");
```

Comment:
```suggestion
    const s = p.replaceAll("+", " ");
```

Reply:
Thank you! I added a test case to prevent this error happening again.